exporters/otlp/include/opentelemetry/exporters/otlp/otlp_grpc_client.h

```diff
@@ -25,12 +29,21 @@ namespace otlp
 
 struct OtlpGrpcClientOptions;
 
+#ifdef ENABLE_ASYNC_EXPORT
+struct OtlpGrpcClientAsyncData;
+#endif
+
 /**
  * The OTLP gRPC client contains utility functions of gRPC.
  */
 class OtlpGrpcClient
 {
 public:
+#ifdef ENABLE_ASYNC_EXPORT
+  OtlpGrpcClient();
+  ~OtlpGrpcClient();
+#endif
```

Comment:
nit - always declare the constructor and destructor, and use the ifdef in the implementation

Reply:
Thanks and one.